core/language/en-GB/EditTemplate.multids

```diff
@@ -21,7 +21,7 @@ Tags/Dropdown/Caption: tag list
 Tags/Dropdown/Hint: Show tag list
 Title/BadCharacterWarning: Warning: avoid using any of the characters <<bad-chars>> in tiddler titles
 Title/Exists/Prompt: Target tiddler already exists
-Title/Relink/Prompt: Update ''<$text text=<<fromTitle>>/>'' to ''<$text text=<<toTitle>>/>'' in the //tags// and //list// fields of other tiddlers
+Title/Relink/Prompt: Update ''<$text text=<<fromTitle>>/>'' to ''<$text text=<<toTitle>>/>'' in the //tags// and //list// fields of other tiddlers, backlinks in <<backlinksLink>> do not change
```

Comment:
Rather than re-using the "Relink" prompt, I think we should add a separate block of text with the warning about references to this tiddler, and therefore a separate translatable string. It might be called `Title/References/Prompt`, and contain the text `The following manual links to this tiddler will not be automatically updated:`.

We don't need the `backlinksLink` macro if we instead directly populate the listing (see below).